cmd/commands/client/command_client.go

```diff
@@ -176,3 +177,15 @@ func (cmd *Command) Kill() error {
 
 	return nil
 }
+
+// TODO this function can be aligned with server function when client and server options will merge into
+func getNetworkDefinition(options CommandOptions) metadata.NetworkDefinition {
+	network := metadata.TestNetworkDefinition
```

Comment:
I would use 'TestNetDefinition', since we talk about specific network name 'TestNet'

Reply:
Fixed